app/scripts/modules/core/projects/configure/configureProject.modal.controller.js

```diff
@@ -37,10 +37,6 @@ module.exports = angular.module('spinnaker.core.projects.configure.modal.control
       pipelines: require('./projectPipelines.modal.html'),
     };
 
-    $timeout(() => {
```

Comment:
This timeout felt odd and its behavior better implemented with done="true" on the templates -- but just wanted to check in case I was missing something.